### PR TITLE
Cypress/E2E: Fix and match with corresponding TM4J IDs

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix and match with corresponding TM4J IDs

### CODE REVIEW

File: e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js [hunk 1]
```diff
@@ -56,36 +56,4 @@ describe('Messaging', () => {
             });
         });
     });
-
-    it('MM-T1797 Compact view: Show single image thumbnail', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js:60]: This is duplicate of https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/messaging/message_spec.js#L168

File: e2e/cypress/integration/messaging/mention_autocomplete_overlap_spec.js [hunk 2]
```diff
@@ -81,12 +81,12 @@ describe('Messaging', () => {
...
     });
 
-    it('MM-T70 At-mention user autocomplete is legible when it overlaps with channel header when drafting a long message containing a file attachment', () => {
+    it('MM-T70_1 At-mention user autocomplete is legible when it overlaps with channel header when drafting a long message containing a file attachment', () => {
         // # Upload file, add message, add mention, verify no overlap
         uploadFileAndAddAutocompleteThenVerifyNoOverlap();
     });
 
-    it('MM-T70 At-mention user autocomplete is legible when it overlaps with channel header when drafting a long message containing a file attachment (1280x900 viewport)', () => {
+    it('MM-T70_2 At-mention user autocomplete is legible when it overlaps with channel header when drafting a long message containing a file attachment (1280x900 viewport)', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/mention_autocomplete_overlap_spec.js:89]: Updated TM4J to reflect and match test steps.

File: e2e/cypress/integration/messaging/system_message_spec.js [hunk 3]
```diff
@@ -60,7 +60,7 @@ describe('System Message', () => {
         });
     });
 
-    it('MM-T213 System message limited options', () => {
+    it('MM-T426 System messages wrap properly', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/system_message_spec.js:63]: I mistakenly update this per my last PR - https://github.com/mattermost/mattermost-webapp/pull/6130/files#diff-41c2ea6f67441c0c6de6e59cd495b80cR63

**Review by srkgupta — APPROVED** [review]
LGTM

**Review by prapti — APPROVED** [review]
LGTM!

**Pull request merged**